README.md

```diff
@@ -1,15 +1,24 @@
-NodePinger is a very simple application that can
-* ping nodes at regular interval;
-* synchronize global node names known to the current node.
+# NodePinger
 
-The list of nodes should be specified in the config.
-```
+A very simple Elixir application that can:
+
+- ping nodes at regular interval;
+- synchronize global node names known to the current node.
+
+## Usage
+
+The list of nodes should be specified in the config:
+
+```elixir
 config: node_pinger, nodes: [:name@host]
 ```
 
-Intervals can be redefined in config (by default, ping is performed every second, synchronize global names is every 5 minutes)
-```
+Intervals can be redefined in the config (by default, ping is performed every second, synchronize global names is every 5 minutes):
+
+```elixir
 config: node_pinger,
-   node_ping_interval: 1000,
```

Comment:
Not sure about the indentation here. Maybe it makes sense in Elixir. Ping me if I wrong and shouldn't have removed those spaces.

Reply:
Hi
It's an excerpt from config/config.exs formatted with `mix format`, so the indentation is the official style.

Reply:
Huh, okay. 

I've rolled the edit back. Thanks!